packages/manager/apps/zimbra/src/pages/dashboard/Organizations/ActionButtonOrganization.tsx

```diff
@@ -12,11 +23,20 @@ export const ActionButtonOrganization: React.FC = () => {
     },
     {
       id: 2,
-      onClick: () => window.open('https://ovhcloud.com', '_blank', 'noopener'),
+      onClick: () => setModalDeleteIsOpen(!modalDeleteIsOpen),
       label: t('zimbra_organization_delete'),
     },
   ];
-  return <ActionMenu items={actionItems} isCompact />;
-};
 
-export default ActionButtonOrganization;
+  return (
+    <>
+      <ActionMenu items={actionItems} isCompact />
+      {modalDeleteIsOpen && (
+        <ModalDeleteOrganization
+          organizationId={organizationItem.id}
+          setIsOpen={() => setModalDeleteIsOpen(!modalDeleteIsOpen)}
+        />
+      )}
```

Comment:
it should be an Outlet and open the modal in a new route

Reply:
It's applied